SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -136,6 +136,21 @@ void USpatialReceiver::OnAddComponent(const Worker_AddComponentOp& Op)
 		return;
 	}
 
+	// Remove all RemoveComponentOps have already been received that have the same entityId and componentId.
```

Comment:
There should be at most one so maybe worth printing a warning if there's more?

Reply:
Sure!